Description: define PATH_MAX when its not defined
 This fixes FTBFS on GNU/Hurd
Author: Mohammed Bilal <rmb@debian.org>
Forwarded: https://github.com/hyperrealm/libconfig/issues/252
Last-Update: 2025-01-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/examples/c/example4.c
+++ b/examples/c/example4.c
@@ -30,6 +30,10 @@
 
 #include <libconfig.h>
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 /* This example reads the configuration file 'example.cfg' and displays
  * some of its contents.
  */
